collectors/python.d.plugin/postgres/postgres.conf

```diff
@@ -103,6 +103,7 @@
 # - 9.5
 # - 9.6
 # - 10
+# - 11
```

Comment:
Maybe something like # - from 9.4 to 13.3

Reply:
Or

 - remove [_Postgres supported versions ..._](https://github.com/netdata/netdata/blob/512f9228ecdd7078f3b26b85700f32553908c522/collectors/python.d.plugin/postgres/postgres.conf#L100-L105) from postgres.conf.
 - add `Postgress v9.4+` to the readme `Requirements` section.